adrs/co2.json

```diff
@@ -0,0 +1,123 @@
+{
+    "stac_version": "1.0.0",
+    "stac_extensions": [],
+    "type": "Feature",
+    "id": "co2",
+    "name": "CO\u2082 (Avg)",
+    "info": "This layer shows the average background concentration of carbon dioxide (CO₂) in our atmosphere for 2020. Redder colors indicate more CO₂. Bluer colors indicate less CO₂.",
+    "bbox": [
+        -180,
+        -90,
+        180,
+        90
+    ],
+    "geometry": {
+        "type": "Polygon",
+        "coordinates": [
+            [
+                [
+                    -180,
+                    -90
+                ],
+                [
+                    180,
+                    -90
+                ],
+                [
+                    180,
+                    90
+                ],
+                [
+                    180,
+                    -90
+                ],
+                [
+                    -180,
+                    -90
+                ]
+            ]
+        ]
+    },
+    "properties": {
```

Comment:
I think the values which are data the dashboard front end is currently dependent on for rendering tiles should either be omitted for now or part of a custom extension, i.e. https://github.com/NASA-IMPACT/covid-api/blob/adrs/stac-co2/adrs/co2.json#L42-L64 and https://github.com/NASA-IMPACT/covid-api/blob/adrs/stac-co2/adrs/co2.json#L93-L122 should be omitted or use a key indicating a custom extension like "frontend:s3_location"

Reply:
We now have the [delta-config](https://user-images.githubusercontent.com/10407788/129632282-f71e9f45-264c-4882-af28-7062c4e56f25.png) to keep in mind as well. It might help if there is metadata that just doesn't fit into any of the predefined attributes in the stac specification. 

If I understand correctly, delta config will have more ui-specific metadata about thematic areas and groupings of datasets but it seems like it could also house information like the legend colormap. I'm sure we will need to define at least one custom extension along the way, though, so you may end up needing to mock a custom extension for item properties anyway.